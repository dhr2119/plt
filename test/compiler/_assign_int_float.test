int main() {
    int i;
    float f;
    i = 5;
    f = 42.42;
    print_int(i);
    print_float(f);
    return 0;
}
