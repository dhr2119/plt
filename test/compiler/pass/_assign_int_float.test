int g;

int main() {
    int i;
    float f;
    g=2;
    i = 5;
    f = 42.42;
    print_int(g);
    print_int(i);
    print_float(f);
    return 0;
}
