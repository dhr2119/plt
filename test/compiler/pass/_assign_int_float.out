2
5
42.420000
